addons/sys_core/fnc_localStartSpeaking.sqf

```diff
@@ -18,11 +18,14 @@
  */
 #include "script_component.hpp"
 
-private _onRadio = parseNumber(_this select 2);
+private _onRadio = (parseNumber (_this select 2)) == 1;
 private _radioId = _this select 3;
 TRACE_1("LOCAL START SPEAKING ENTER", _this);
+
 ACRE_LOCAL_SPEAKING = true;
-if (_onRadio == 1) then {
+["acre_startedSpeaking", [acre_player, _onRadio == 1, _radioId]] call CBA_fnc_localEvent; // [unit, on radio, radio ID]
```

Comment:
I think `_onRadio ` is already a bool?

Reply:
Yeah, missed that one when I changed it to bool above, thanks!